save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -22,9 +22,10 @@ import okio.Path.Companion.toPath
 
 /**
  * A plugin that runs an executable on a file and compares output with expected output.
+ * @property testConfig
  */
 @Suppress("INLINE_CLASS_CAN_BE_USED")
-class FixPlugin : Plugin {
+class FixPlugin(override val testConfig: TestConfig) : Plugin(testConfig) {
```

Comment:
```suggestion
class FixPlugin(testConfig: TestConfig) : Plugin(testConfig) {
```
Is it useful to override it here, or we can just use the property from superclass?

Reply:
yeah, I just wanted to make a small defence here for cases if we will use this value somewhere else, fixed